### PR TITLE
refs #32516 - add rocky linux to redhat family

### PR DESCRIPTION
**Pull request by sbernhard**

### CODE REVIEW

File: app/models/operatingsystem.rb [hunk 1]
```diff
@@ -59,7 +59,7 @@ class Operatingsystem < ApplicationRecord
   scoped_search :relation => :provisioning_templates, :on => :name, :complete_value => :true, :rename => "template", :only_explicit => true
 
   FAMILIES = { 'Debian'    => %r{Debian|Ubuntu}i,
-               'Redhat'    => %r{RedHat|Centos|Fedora|Scientific|SLC|OracleLinux|AlmaLinux}i,
+               'Redhat'    => %r{RedHat|Centos|Fedora|Scientific|SLC|OracleLinux|AlmaLinux|Rocky}i,
```
Comment (lzap) [inline app/models/operatingsystem.rb:62]: This hightlights one inconsistency we have - some OSes have "Linux" suffix however others do not. Would you mind removing all suffixes from here?

```suggestion
               'Redhat'    => %r{RedHat|Centos|Fedora|Scientific|SLC|Oracle|Alma|Rocky}i,
```

Also add all other variants to all templates too: Scientific, Oracla, Alma, Rocky so they are all correctly associated.

Finaly, could you please also rename almalinux to alma in a regular expression in app/helpers/operatingsystems_helper.rb ?

Comment (ekohl) [reply app/models/operatingsystem.rb:62]: Please don't do this. This is how it's identified by facts and they should be as close as possible to what's reported for a precise match.

**Review by lzap — APPROVED** [review]
Although I do not see any reason why we need to stick with Puppet Facter, I do not insist.

**Comment by ekohl** [comment]
The Docker containers are also named [oraclelinux](https://index.docker.io/_/oraclelinux) and [almalinux](https://index.docker.io/_/almalinux) respectively.

**Comment by ezr-ondrej** [comment]
CP 2.5: #8587

**Pull request merged**